docker/nginx/conf.d/default.conf

```diff
@@ -2,34 +2,37 @@ server {
     root /srv/sylius/public;
 
     location / {
-        # try to serve file directly, fallback to app.php
-        try_files $uri /app.php$is_args$args;
-    }
+            # try to serve file directly, fallback to index.php
+            try_files $uri /index.php$is_args$args;
+        }
 
-    # PROD
-    location ~ ^/app\.php(/|$) {
-        # Comment the next line and uncomment the lines after to enable dynamic resolution (incompatible with Kubernetes)
-        fastcgi_pass php:9000;
-        #resolver 127.0.0.11;
-        #set $upstream_host php;
-        #fastcgi_pass $upstream_host:9000;
+        # DEV
+        # This rule should only be placed on your development environment
+        # In production, don't include this and don't deploy app_dev.php or config.php
+        location ~ ^/index\.php(/|$) {
+            fastcgi_pass php:9000;
+            fastcgi_split_path_info ^(.+\.php)(/.*)$;
+            include fastcgi_params;
 
-        fastcgi_split_path_info ^(.+\.php)(/.*)$;
-        include fastcgi_params;
-        # When you are using symlinks to link the document root to the
```

Comment:
I'm not a big fan of removing this. Ok to comment, but not to remove.
It's important to deal with all cases, and in production the application is often a symlink.